src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb

```diff
@@ -186,7 +186,7 @@ Public MustInherit Class BasicTestBase
 
         Dim assemblyName As String = Nothing
         Dim sourceTrees = ParseSourceXml(source, parseOptions, assemblyName)
-        Dim compilation = CreateEmptyCompilation(sourceTrees, allReferences, options, assemblyName)
+        Dim compilation = CreateEmptyCompilation(sourceTrees.ToArray(), allReferences, options, assemblyName:=assemblyName)
```

Comment:
>.ToArray() [](start = 60, length = 10)

Is `.ToArray()` necessary here?

Reply:
Yes. There is no way to do an implicit conversion from `IEnumerable(Of SyntaxTree)` to `BasicTestSource` as we don't allow widening conversions on interfaces. 

---
In reply to: [192156452](https://github.com/dotnet/roslyn/pull/25130#discussion_r192156452) [](ancestors = 192156452)